### DISCUSSION TITLE
Can't save a webpage with pop up mini boxes properly

### DISCUSSION THREAD

**Discussion by georg122** [gildas-lormeau/SingleFile#1550] [Q&A]
I'm trying to save medical records because they only let you search for up to one year of your records.
The webpage is https://www.gesy.org.cy/launchpad.html and after you click "beneficiary portal"
Although you need a password and username to log in. 

There are small arrows "->" which when you click on them shows more details like the doctor's examination in a pop-up box
I tried to save the whole page, but those pop up boxes do not open up in the saving (also before saving the page I checked with the network tab on dev options and didn't see activity when you click on an arrow "->". Even if I open them before saving, the page still won't open after the page is saved. I tried a few settings like to save the scripts but still nothing. 
I'm not sure what else to do.

Here is how the webpage looks like in the first image and in the second image it shows how the pop up box is.

![web1](https://github.com/user-attachments/assets/74f5d48a-9ee0-4f1d-9f41-3ff3a350ee3e)
![web2](https://github.com/user-attachments/assets/0c1d78c7-a6b3-4eab-bee1-fbd5ecb3c64f)

**Comment by gildas-lormeau** [comment]
Could you please unzip the attached file, import the settings file in SingleFile via the options page, save the page and tell me if it fixes the issue?

[singlefile-settings.zip](https://github.com/user-attachments/files/16788190/singlefile-settings.zip)

**Reply by georg122** [reply]
> Could you please unzip the attached file, import the settings file in SingleFile via the options page, save the page and tell me if it fixes the issue?
> 
> [singlefile-settings.zip](https://github.com/user-attachments/files/16788190/singlefile-settings.zip)

I tried it but despite the html file size which is bigger, i get a blank screen when I open the file

**Reply by georg122** [reply]
@gildas-lormeau any help on this?

**Reply by grahamperrin** [reply]
Is Google Gemini similarly affected? 

With the original page, clicking a link presents a pop-up, then moving the pointer over the pop-up highlights the relevant section of text: 

<img width="786" height="135" alt="image" src="https://github.com/user-attachments/assets/962b8fcd-ae36-4028-a8d8-b45aa516a928" />

* and from there, the link can be followed. 

With the saved file, the initial click is ineffective: 

<img width="786" height="176" alt="image" src="https://github.com/user-attachments/assets/e4a4dabd-e1bc-4f94-953c-6dc486c4f652" />
